.github/workflows/coverage.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       - name: Prepare
         run: .github/scripts/run-prepare ${{ matrix.buildsystem }} ${{ matrix.os }}
       - name: Test
+        if: github.repository == "jupp0r/prometheus-cpp"
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
```

Comment:
Maybe it would be cleaner to move this env block up to the job level and check for the presence of the `COVERALLS_REPO_TOKEN` variable in the environment.

Reply:
Yeah I thought so too but if conditions don't work on jobs :(